### DISCUSSION TITLE
Cant download .exe version

### DISCUSSION THREAD

**Discussion by eagle2442** [mircokroon/minecraft-world-downloader#267] [Q&A]
I get this error message when I try to download 
![Screenshot 2021-12-13 174407](https://user-images.githubusercontent.com/96089035/145900877-6fd96a4c-64c9-4873-9760-53973c5e5fc1.png)

**Comment by eagle2442** [comment]
Forgot to add, I have Java version 8 downloaded and I think im up to date

**Reply by mircokroon** [reply]
As stated in the readme, the minimum version is 16 (soon to be 17), not 8.

**Reply by eagle2442** [reply]
Oh ok. Could you send me the link to java 16, and if i download it will it automatically replace 8

**Reply by rautamiekka** [reply]
You need to manually uninstall older versions (for ex, Java 8 -> Java 16) if you don't need the older one, but an update (for ex, Java 8 Update 1 -> Update 2, or Java 17.0.0 -> 17.0.1) will overwrite an older one.

For Window$ (and maybe Mac, but not Linux):
- The newest=https://www.oracle.com/java/technologies/downloads/
- Java 16 (scroll down, the page top is confusingly worded)=https://www.oracle.com/java/technologies/javase/jdk16-archive-downloads.html
